Missionframework/kp_liberation_config.sqf

```diff
@@ -611,12 +611,12 @@ box_transport_config = [
     ["rhsusf_M1083A1P2_D_fmtv_usarmy", -5.0, [0,-0.2,0.45], [0,-1.9,0.45]],
     ["rhsusf_M1083A1P2_WD_flatbed_fmtv_usarmy", -5.0, [0,-0.2,0.45], [0,-1.9,0.45]],
     ["rhsusf_M1083A1P2_WD_fmtv_usarmy", -5.0, [0,-0.2,0.45], [0,-1.9,0.45]],
-    ["rhsusf_M977A4_BKIT_M2_usarmy_d", -6.5, [0,0.4,0.7], [0,-1.3,0.7], [0,-3,0.7]],
-    ["rhsusf_M977A4_BKIT_M2_usarmy_wd", -6.5, [0,0.4,0.7], [0,-1.3,0.7], [0,-3,0.7]],
-    ["rhsusf_M977A4_BKIT_usarmy_d", -6.5, [0,0.4,1.4], [0,-1.3,1.4], [0,-3,1.4]],
-    ["rhsusf_M977A4_BKIT_usarmy_wd", -6.5, [0,0.4,1.4], [0,-1.3,1.4], [0,-3,1.4]],
-    ["rhsusf_M977A4_usarmy_d", -6.5, [0,0.4,1.4], [0,-1.3,1.4], [0,-3,1.4]],
-    ["rhsusf_M977A4_usarmy_wd", -6.5, [0,0.4,1.4], [0,-1.3,1.4], [0,-3,1.4]],
+	["rhsusf_M977A4_BKIT_M2_usarmy_d", -6.5, [0,0.4,0], [0,-1.3,0], [0,-3,0]],
+	["rhsusf_M977A4_BKIT_M2_usarmy_wd", -6.5, [0,0.4,0], [0,-1.3,0], [0,-3,0]],
+    ["rhsusf_M977A4_BKIT_usarmy_d", -6.5, [0,0.4,0.6], [0,-1.3,0.6], [0,-3,0.6]],
+	["rhsusf_M977A4_BKIT_usarmy_wd", -6.5, [0,0.4,0.6], [0,-1.3,0.6], [0,-3,0.6]],
+	["rhsusf_M977A4_usarmy_d", -6.5, [0,0.4,0.7], [0,-1.3,0.7], [0,-3,0.7]],
+	["rhsusf_M977A4_usarmy_wd", -6.5, [0,0.4,0.7], [0,-1.3,0.7], [0,-3,0.7]],
```

Comment:
Spaces should be used for indentation in this file. There are tabs here.

Reply:
Ah, okay, I see. I thougt spaces were my mishap.